wormhole-connect-loader/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@wormhole-foundation/wormhole-connect",
-  "version": "0.0.12",
+  "version": "0.1.3",
```

Comment:
did you mean 0.0.13?

Reply:
The latest on npm is [0.1.3-beta.0-development](https://www.npmjs.com/package/@wormhole-foundation/wormhole-connect/v/0.1.3-beta.0-development)

Reply:
weird diff then but wfm!